src/test/java/seedu/address/logic/commands/HelpCommandTest.java

```diff
@@ -11,14 +12,40 @@
 import seedu.address.ui.testutil.EventsCollectorRule;
 
 public class HelpCommandTest {
+
     @Rule
     public final EventsCollectorRule eventsCollectorRule = new EventsCollectorRule();
 
     @Test
     public void execute_help_success() {
-        CommandResult result = new HelpCommand().execute();
-        assertEquals(SHOWING_HELP_MESSAGE, result.feedbackToUser);
+        HelpCommand command = new HelpCommand();
+        assertCommandSuccess(command, SHOWING_HELP_MESSAGE);
         assertTrue(eventsCollectorRule.eventsCollector.getMostRecent() instanceof ShowHelpRequestEvent);
         assertTrue(eventsCollectorRule.eventsCollector.getSize() == 1);
     }
+
+    @Test
+    public void execute_helpForTask_success() {
+        HelpCommand command = new HelpCommand("task");
+        assertCommandSuccess(command, TaskCommand.MESSAGE_USAGE);
+    }
+
+    @Test
+    public void execute_helpForDelete_success()    {
```

Comment:
Change this to execute_helpForRemove_success()

In fact, change all Delete to Remove